code/modules/scripting/Implementations/Telecomms.dm

```diff
@@ -208,6 +211,9 @@
 		/* sanitize EVERYTHING. fucking players can't be trusted with SHIT */
 
 		signal.data["message"] 	= interpreter.GetCleanVar("$content", signal.data["message"])
+		if (italicized)
+			// If the original message was italicized, we need to italicize it again.
+			signal.data["message"] = "<i>" + signal.data["message"] + "</i>";

```

Comment:
Minor nitpick, but ideally this would be ```"<i>[signal.data["message"]]</i>"``` because that's faster than string concatenation. (best to use ```text("<i>[]</i>", signal.data["message"])``` to avoid weirdness with those square brackets.)